conf/app.conf

```
@@ -1,5 +1,6 @@
 appname = sga_tercero_mid
-httpport = ${SGA_TERCERO_MID_HTTP_PORT}
+# httpport = ${SGA_TERCERO_MID_HTTP_PORT}
+httpport = 8181
```

```suggestion
httpport = ${SGA_TERCERO_MID_HTTP_PORT}
```
Mantener como estaba